wolfssl/wolfcrypt/wc_port.h

```diff
@@ -593,6 +594,9 @@ WOLFSSL_API int wolfCrypt_Cleanup(void);
         #include <dirent.h>
         #include <unistd.h>
         #include <sys/stat.h>
+        #define XWRITE      write
```

Comment:
Not available on all platforms.
Is the BIO code going to compile on Windows?

Reply:
`XWRITE` is excluded from Windows builds. I just built wolfSSL with libest support on Windows and successfully ran the testsuite tests.